includes/db/Database.php

```diff
@@ -202,8 +202,8 @@ function getServerInfo();
  */
 abstract class DatabaseBase implements DatabaseType {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "DatabaseBase" has 159 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)

[project-id]: wikia:app